src/db/controller/packet.ts

```
@@ -19,17 +19,20 @@ import { PacketFeeController } from './packetFee'
 export class PacketController {
   public static tableNamePacketSend = 'packet_send'
   public static tableNamePacketTimeout = 'packet_timeout'
   public static tableNamePacketWriteAck = 'packet_write_ack'
+  private static logger = createLoggerWithPrefix('[PacketController] ');
 
   public static async feedEvents(
     rest: RESTClient,
     chainId: string,
     events: PacketEvent[]
   ): Promise<() => void> {
+    this.logger.info(`feedEvents: chainId=${chainId}, events.length=${events.length}`);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)